### PR TITLE
[Tables] Update from testframework -> recording

### PR DESCRIPTION
**Pull request by seankane-msft**

### CODE REVIEW

File: sdk/tables/aztable/zt_tableRecordedTests.go [hunk 1]
```diff
@@ -63,27 +63,27 @@ func recordedTestSetup(t *testing.T, testName string, endpointType EndpointType,
 	assert := assert.New(t)
 
 	// init the test framework
-	context := testframework.NewTestContext(func(msg string) { assert.FailNow(msg) }, func(msg string) { t.Log(msg) }, func() string { return testName })
-	recording, err := testframework.NewRecording(context, mode)
+	context := recording.NewTestContext(func(msg string) { assert.FailNow(msg) }, func(msg string) { t.Log(msg) }, func() string { return testName })
+	recordingInstance, err := recording.NewRecording(context, mode)
```
Comment (christothes) [inline sdk/tables/aztable/zt_tableRecordedTests.go:67]: nit: This bight be more idiotmatic and less verboase
```suggestion
	r, err := recording.NewRecording(context, mode)
```

**Review by christothes — APPROVED** [review]

**Review by catalinaperalta — APPROVED** [review]
LGTM. Although a nit not related to this PR but maybe you could rename `zt_tableRecordedTests.go` to `zt_table_recorded_tests.go`

**Pull request merged**